build.gradle

```diff
@@ -286,10 +289,13 @@ project('serializers:protobuf') {
         compile project(':serializers:shared')
         compile group: 'com.google.protobuf', name: 'protobuf-java', version: protobufProtocVersion
         compile group: 'com.google.protobuf', name: 'protobuf-java-util', version: protobufUtilVersion
+        compileOnly group: 'io.pravega', name: 'pravega-client', version: pravegaVersion
```

Comment:
Same comment as above, here and other serializer packages below.

Reply:
do review this in PR #160 
i have reverted the changes from this PR